openedx/core/djangoapps/courseware_api/views.py

```diff
@@ -623,8 +622,8 @@ def get(self, request, course_key_string, *args, **kwargs):  # lint-amnesty, pyl
             resp['unit_id'] = str(path[3])
             resp['block_id'] = str(block_key)
 
-        except UnavailableCompletionData:
-            pass
+        except (ItemNotFoundError, NoPathToItem, UnavailableCompletionData):
+            pass  # leaving all the IDs as None indicates a redirect to the first unit in the course, as a backup
```

Comment:
This change was because of how the MFE works.

With the other change in this PR - the MFE will now see a `resume_url` like `http://learning.edx.org/course/course-v1:xxx` (just a bare path). That in turn will call this `/api/courseware/resume/{course_key}` view. Which looks up the resume block again and tries to offer that up.

Before this change here, if it found an invalid resume block, it would error out. Now it tells the MFE to go to the beginning of the course.